doc/sphinx-guides/source/admin/metadatacustomization.rst

```diff
@@ -293,9 +295,9 @@ FieldType definitions
 +---------------+------------------------------------+
 | text          | Any text other than newlines may   |
 |               | be entered into this field.        |
-|               | The text fieldtype can be used     |
-|               | combined with                      |
-|               | #controlledVocabulary as boolean.  |
+|               | The text fieldtype may used to     |
```

Comment:
@luddaniel There's just a missing "be" between may and used. Could you add that?

Reply:
@jggautier whoops, that was my bad, part of my rewrite. I just fixed it in 649e2afe76.

Reply:
Cool thanks. That was my only suggestion for the changes in the documentation. I'll remove myself from the Assignment list on this PR.